source/whats-new.txt

```diff
@@ -12,3 +12,250 @@ What's New
    :depth: 1
    :class: singlecol
 
+Learn what's new in:
+
+* :ref:`Version 1.10 <version-1.10>`
+* :ref:`Version 1.9 <version-1.9>`
+* :ref:`Version 1.8 <version-1.8>`
+* :ref:`Version 1.7 <version-1.7>`
+* :ref:`Version 1.6 <version-1.6>`
+* :ref:`Version 1.5 <version-1.5>`
+* :ref:`Version 1.4 <version-1.4>`
+* :ref:`Version 1.3 <version-1.3>`
+* :ref:`Version 1.2 <version-1.2>`
+* :ref:`Version 1.1 <version-1.1>`
+* :ref:`Version 1.0 <version-1.0>`
+
+
+.. _version-1.10.0:
+.. _version-1.10:
+
+What's New in 1.10
+--------------------
+
+.. important:: Upgrade to Version 1.10.1 or Higher 
+
+  The 1.10.1 Go driver patches a bug that can cause data corruption when 
+  rotating :ref:`Data Encryption Keys <csfle-key-architecture>` (DEKs) encrypted 
+  with a :ref:`Customer Master Key <csfle-key-architecture>` hosted on Google 
+  Cloud Key Management Service or Azure Key Vault.
+
+New features of the 1.10 Go driver release include: 
+
+- Added full support for MongoDB 6.0.
+  
+- Support for new features related to :ref:`qe-manual-feature-qe`,
```

Comment:
Issue: The features in all of these versions waver between starting with a verb and starting with a noun. For consistency, it's probably best if we choose one or the other. 

Suggestion: Remove "Added", "Included", or other unnecessary verbs at the beginnings of these items.

Reply:
Noted and fixed!